bank2ynab.conf

```diff
@@ -2,7 +2,7 @@
 
 [DEFAULT]
 # Specify where you save your downloaded CSV files:
-Source Path = %USERPROFILE%\Downloads
+Source Path = /home/jacob/Downloads/
```

Comment:
I think we want to keep the generic user here. Not everyone has the same first name.

Reply:
:+1: then again not everybody uses windows so it should perhaps be some standard pythonpath https://stackoverflow.com/questions/35851281/python-finding-the-users-downloads-folder seems to suggest that it is possible to do.

Reply:
That's what the find_directory function in b2y_utilities does if needed. If Source Path is blank, it uses that.